docs/development-guide.md

```diff
@@ -44,6 +44,26 @@ To build ceph-csi in a container:
 
 The built binary will be present under `_output/` directory.
 
+### Running Ceph-CSI tests
+
+Once the changes to the sources compile, it is good practise to run the tests
+that validate the style and other basics of the source code. Execute the unit
+tests (in the `*_test.go` files) and check the formatting of YAML files and
```

Comment:
it also validates the .md files

Reply:
yes, and more. But I added "MarkDown documents" too now.